python/hsfs/feature_view.py

```diff
@@ -319,6 +439,36 @@ def create_training_data(
     ):
         """Create a training dataset and save data into `location`.
 
+        !!! example "Create random splits"
+            ```python
+            feature_view = fs.get_feature_view(name="feature_view_name")
+
+            version, job = feature_view.create_training_data(
+                description='transactions_dataset_jan_feb',
+                data_format='csv',
+                write_options={"wait_for_job": False}
+            )
+            ```
+
+        !!! example "Create time-series based splits"
+            ```python
+            from datetime import datetime
+            date_format = "%Y-%m-%d %H:%M:%S"
+
+            start_time = int(float(datetime.strptime("2022-05-01 00:00:00", date_format).timestamp()) * 1000)
+            end_time = int(float(datetime.strptime("2022-06-04 23:59:59", date_format).timestamp()) * 1000)
```

Comment:
use simpler date formats. For example just string `2022-05-01` works now. Also python datetime objects works as well